src/screens/EventView.js

```diff
@@ -1,27 +1,79 @@
 import React, { Component } from "react";
 import map from "../assets/images/map.PNG";
+import addpicture from "../assets/images/addpicture.png";
+import sharedStyles from "../assets/CSS/sharedStyles.css"
 
 class EventView extends Component {
+   constructor() {
```

Comment:
If you're going to have a constructor then we should pass the props object even if we aren't going to use it